Dshop/apps/products_catalogue/models.py

```diff
@@ -27,3 +27,17 @@ class Meta:
         verbose_name = 'Category'
         verbose_name_plural = 'Categories'
 
+
+class Product(models.Model):
+    name = models.CharField(max_length=255)
+    category = models.ForeignKey(Category, on_delete=models.CASCADE)
+    price = models.DecimalField(max_digits=10, decimal_places=2)
+    price_last_30_days = models.DecimalField(max_digits=10, decimal_places=2, null=True, blank=True)
+    short_description = models.TextField()
+    full_description = models.TextField()
+
```

Comment:
I suggest to add additional fields:
- discounted_price  or discount (relation to separate table "discount")
- created_at
- updated_at
- properties - a relation to table contains details (weight, dimensions, color, waranty and so on)

Reply:
Please, do not overengineer :-) 
Yet :-) 
Let's stick to plan. Variants, etc, in next steps. 
Agreed on adding created_at and updated_at.